src/main.js

```diff
@@ -4,3 +4,8 @@ export default placeholder;
 
 // Dialogs
 export { default as ModalDialog } from './components/ModalDialog';
+
+export { default as Label } from './components/Label';
+
+// Layouts
+export { default as StackLayout } from './layouts/StackLayout';
```

Comment:
No need to import/export components from main.js
we should import and use them directly in the components when we need

Reply:
and also please remove ModalDialog

Reply:
Removed all exports from main.js